src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,31 @@
+package lotto.model;
+
+import java.util.List;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        if (numbers.size() != 6) {
+            throw new IllegalArgumentException("[ERROR] 로또 번호는 6개여야 합니다.");
```

Comment:
사용되는 메세지나 숫자들을 상수화하면 더 가독성이 좋을 것 같아요!

Reply:
이건 스타일의 차이인 것 같은데 저는 if문이 validate안에 있는 것보다 if문을 다 메서드로 분리해서 메서드를 호출하는 게 좀 더 깔끔한 것 같더라구요! 아래처럼요!

```suggestion
private void validate(List<Integer> numbers) {
    checkSize(numbers);
    ...
}
...
private void checkSize(List<Integer> numbers) {
    if (numbers.size() != 6) {
         throw new IllegalArgumentException("[ERROR] 로또 번호는 6개여야 합니다.");
}
```